src/__tests__/parser.test.ts

```diff
@@ -85,4 +90,13 @@ describe('parser', () => {
             expect(convertEmojiTags('[!WARN]')).toBe('[!WARN]');
         });
     });
+
+    describe('truncateString', () => {
+        it('cuts a string down to the discord character limit and corrects missing codeblock backticks if needed', () => {
+            const section = getSection("Example", substrFile);
+            const truncatedStr = truncateString(section);
+            expect(truncatedStr).toHaveLength(1024);
+        })
+    })
```

Comment:
So don't feel obligated to make or improve any tests, but figured I would leave a comment here just in case you were curious.

This test doesn't verify that backticks were added. I think the easiest way to accomplish a more robust test is to use `.toMatchSnapshot()`. When this is run via `npm run test -- -u` it will create a snapshot file which contains the received value of the expect. It will automatically pass. Then you can check the snapshot output to ensure it contains the backticks you expect. From there, if anyone breaks the function, the snapshot will no longer match and the test will fail. Happy to chat more about snapshot testing in detail, but I think it is the fastest way to verify what we want. 

```suggestion
    describe('truncateString', () => {
        it('cuts a string down to the discord character limit and corrects missing codeblock backticks if needed', () => {
            const section = getSection("Example", substrFile);
            const truncatedStr = truncateString(section);
            expect(truncatedStr).toHaveLength(1024); // might need to update this after making the max length an input param
            expect(truncatedStr).toMatchSnapshot();
        })
    })
```

Reply:
Sounds good. As you mentioned, I used the `truncateString` test to debug rather than cover code. After finding the solution and removing any logs, the only thing that remained was the simple length check and I figured I would leave that in.

Definitely curious to improve this test in particular to specifically test for missing backticks, but I think I'll leave that to another PR or "ticket", as that will probably take me a couple more days.

Reply:
Awesome, sounds good to me!